xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/jackson/modules/AccountTransactionsTransactionDeserializer.java

```diff
@@ -57,10 +59,12 @@ public AccountTransactionsTransaction<?> deserialize(
     Transaction transaction = objectMapper.readValue(node.toString(), Transaction.class);
     long ledgerIndex = node.get("ledger_index").asLong(-1L);
     String hash = node.get("hash").asText();
+    UnsignedLong date = node.has("date") ? UnsignedLong.valueOf(node.get("date").asLong()) : null;
```

Comment:
```suggestion
    Optional<UnsignedLong> closeDate = Optional.ofNullable(node)
      .filter($ -> $.has("date"))
      .map($ -> $.get("date"))
      .map(JsonNode::asLong)
      .map(UnsignedLong::valueOf);
```

Reply:
`node.get("date")` will return `null` if `date` doesn't exist, so I changed this to
```
Optional<UnsignedLong> closeDate = Optional.ofNullable(node.get("date"))
      .map(JsonNode::asLong)
      .map(UnsignedLong::valueOf);
```

Reply:
Awesome, nice catch!